lib/aika_web/router.ex

```diff
@@ -46,11 +47,14 @@ defmodule AikaWeb.Router do
     post "/users/invite", UserController, :invite
     delete "/users/:id", UserController, :delete
     get "/users/:id", UserController, :show
+    get "/users/:id/generate_api_token", UserController, :generate_api_token
```

Comment:
I don't think we need the users id here as we would only want the users to be able to regenerate their own API token, that way we won't have to validate that the current user is the requesting user in the controller.

Also as this change the user record I think it should be a POST instead of a GET